<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/mainIcon.png</normaloff>icon/mainIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,5,1">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="selectBtn">
            <property name="text">
             <string>SELECT</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icon/icon/select.png</normaloff>:/icon/icon/select.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoomBtn">
            <property name="text">
             <string>ZOOM</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icon/icon/search.png</normaloff>:/icon/icon/search.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_rotate_right">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icon/icon/rotateR.png</normaloff>:/icon/icon/rotateR.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_rotate_left">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icon/icon/rotateL.png</normaloff>:/icon/icon/rotateL.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editDialogBtn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icon/icon/edit.png</normaloff>:/icon/icon/edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fitBtn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icon/icon/home.png</normaloff>:/icon/icon/home.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionsave"/>
    <addaction name="actionload"/>
    <addaction name="actiontray"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="output">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>190</width>
          <height>359</height>
         </rect>
        </property>
        <attribute name="label">
         <string>tools</string>
        </attribute>
        <widget class="QPushButton" name="btn_result">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>11</y>
           <width>161</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>RESULT</string>
         </property>
        </widget>
        <widget class="QPushButton" name="designBtn">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>47</y>
           <width>161</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>DESIGN</string>
         </property>
        </widget>
        <widget class="QPushButton" name="resetBtn">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>83</y>
           <width>161</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>REST</string>
         </property>
        </widget>
        <widget class="QToolButton" name="showoutBtn">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>119</y>
           <width>161</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>SHOW OUTPUT</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="selectGroup">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>185</y>
           <width>161</width>
           <height>79</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="elementSelect">
            <property name="text">
             <string>ELEMENT</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="nodeSelect">
            <property name="text">
             <string>NODE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="setting">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>170</width>
          <height>359</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 2</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>LINE WIDTH</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="lineWidthSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>NODE SIZE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="nodeSizeSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>MAGNIFY SIZE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="magnifySilder">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>DISPLACEMENT SIZE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="displacementSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="showDisBtn">
           <property name="text">
            <string>SHOW  DISP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="configureBtn">
           <property name="text">
            <string>REPAINT</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="view">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>278</width>
          <height>359</height>
         </rect>
        </property>
        <attribute name="label">
         <string>页</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>STRESS GRAPH</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGraphicsView" name="strainView"/>
         </item>
         <item>
          <widget class="QSlider" name="viewSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>92</width>
          <height>359</height>
         </rect>
        </property>
        <attribute name="label">
         <string>EXPLORING</string>
        </attribute>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>0</y>
           <width>69</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>LAB</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cmdText">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QToolButton" name="chooseBtn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>27</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>S</string>
         </property>
        </widget>
        <widget class="QPushButton" name="executeBtn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>93</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>EXECUTE</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionload">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actiontray">
   <property name="text">
    <string>tray</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
